.idea/codeStyles/Project.xml

```diff
@@ -246,4 +246,4 @@
       <option name="ENUM_CONSTANTS_WRAP" value="5" />
     </codeStyleSettings>
   </code_scheme>
-</component>
\ No newline at end of file
+</component>
```

Comment:
Can we remove this change too? 🙈

Reply:
I don't know why it keeps doing this, I was certain I updated my settings to avoid this but now this one is a weird one for the IDE to make.

Reply:
Yeah it's super annoying. I'd suggest just throwing it away with your git client – I've had to do that a lot in the last few builds, too 🤔